Source/Task/XTaskQueuePriv.h

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#include <httpClient/async.h>
+#include "XTaskQueue.h"
```

Comment:
revert this one. Matt made this change for xplat work

Reply:
This header is shared with Windows though -- there is no httpClient there.  Was this one done wrong?

Reply:
Brian's change is correct